docs/codingconventions.adoc

```diff
@@ -21,8 +31,75 @@ It is not an exhaustive list and it will be updated as we go. In the mean time,
 - Nested modules should exist under the `modules/` folder
 - Examples of using the module should exist under the `examples/` folder and have a README to explain the goal and usage of the example
 
-== Case type and Names
+== Documentation format
+
+We use https://asciidoc.org/[AsciiDoc] with the `*.adoc` file extension. The only exception is for the README files that need to be displayed on the Terraform Registry.
+
+[NOTE]
+====
+As GitHub renders AsciiDoc files using https://asciidoctor.org/[AsciiDoctor], it is the current reference regarding AsciiDoc syntax we use in the documentation.
+====
+
+Terraform Registry currently do not support AsciiDoc,so documentation content such as `README` files must be in Markdown format.
```

Comment:
Try: "As the Terraform Registry does not support AsciiDoc, the `README.md` and certain other documentation must be available in Markdown format,"